github/github.go

```diff
@@ -487,7 +487,7 @@ func (r RateLimits) String() string {
 	return Stringify(r)
 }
 
-// RateLimit is deprecated.  Use RateLimits instead.
+// RateLimit is deprecated.  Deprecated: Use RateLimits instead.
```

Comment:
yeah, according to golang/blog@257114a it has to appear at the beginning of the paragraph.